client/src/main/webapp/js/custom/services/authenticationService.js

```diff
@@ -25,11 +25,17 @@ angular.module('app.services').service('authenticationService', ['$http', '$q',
     this.logout = function () {
         var d = $q.defer();
 
-        $http.post(propertiesConstant.RESTFUL_API_URL + '/api/authentication/logout', null)
+        var authToken = storageService.getLocalItem(storageConstant.AUTH_TOKEN);
```

Comment:
Why local storage rather then session storage, session storage will ensure the credentials are lost when the browser is closed.